test/e2e/Dockerfile.dockerignore

```
@@ -0,0 +1,12 @@
+# This Dockerfile uses the repository root as its build context, but additionally
+# needs the E2E test package that is intentionally absent from the production
+# image context.
+**
```

Hadolint (2.14.0)

[error] unexpected '*'
expecting '#', '\', ADD, ARG, CMD, COPY, ENTRYPOINT, ENV, EXPOSE, FROM, HEALTHCHECK, LABEL, MAINTAINER, ONBUILD, RUN, SHELL, STOPSIGNAL, USER, VOLUME, WORKDIR, a pragma, at least one space, or end of input

(DL1000)